draft-ietf-teas-actn-poi-applicability.md

```diff
@@ -2267,20 +2487,20 @@ might be a 100Gb/s trunk port shared by ten 10GE client ports.
 
 The controller, when asked to set a 10GE client connectivity service
 between two muxponder's client ports, needs first to check whether
-there is already an existing WDM tunnel between the two muxponders
+there is already an existing DWDM tunnel between the two muxponders
 and then take different actions:
 
-1. if the WDM tunnel already exists, the controller needs only to
+1. if the DWDM tunnel already exists, the controller needs only to
```

Comment:
I-D.ietf-ccamp-wdm-tunnel-yang defines the 'WDM tunnel' model (and not the 'DWDM tunnel' model)

```suggestion
1. if the WDM tunnel already exists, the controller needs only to
```